stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>This is a really great story. I particularly love the way you lead the reader through the process, building up your model and refining it as you go. It's really informative, and all of the code and examples are very clear.</p><p> </p><p>I would second what Jack suggested at the outset: Overwatch is a beatuiful and visual game, so including some screenshots, map designs or even videos would make the story more visually lively (personally I'd scatter them througout if you can find a justification). Apart from being beautiful, I think it's also hard to really understand the game without seeing a bit of what it looks like (is it like chess, like football, like Mario, etc. You do mention at the outset that it's an FPS, but that may not b obvious to everyone).</p><p> </p><p>I generally agreed with Jack's comments (all very helpful).</p><p> </p><p>You also have some really great conclusions; they come across as intersting but balanced. I'd be curious to hear of ways to improve the results using Overwatch-specific data. E.g. did you consider looking at game type, or hero loadouts as predictors?</p><p> </p><p>If you include a video, please let it be of the authors playing a game ;)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

Reply:
<p>Yeah, we considered especially game type. I think we even tried some quick analyses, like win rate by game type or something, but nothing seemed to yield much predictive power. Hero loadouts we didn't look into, the data set we had didn't include that. I added a sentence to conclusions about this.</p>